### PR TITLE
fix: Can't use HTML code on TS files like that

### PR DESCRIPTION
**Pull request by Dacaramo**
You can't use HTML tags in TS code like that, you will always get an error, please accept this changes so the documentation is correct. Thank you!

### CODE REVIEW

File: docs/pages/docs/usage/configuration.mdx [hunk 1]
```diff
@@ -365,13 +365,14 @@ To achieve consistent usage of translation values and reduce redundancy, you can
...
 import {getRequestConfig} from 'next-intl/server';
+import { createComponent, ReactNode } from 'react;
 
 export default getRequestConfig(async ({locale}) => {
   // ...
 
   return {
     defaultTranslationValues: {
-      important: (chunks) => <b>{chunks}</b>,
+      important: (chunks) => createComponent('b', null, chunks) as ReactNode,
```
Comment (A7med3bdulBaset) [inline docs/pages/docs/usage/configuration.mdx:375]: Typescript supports JSX e.g. `<b>{chunks}</b>`>.

Just rename your file to be `i18n.tsx` not `.ts`

File: docs/pages/docs/usage/configuration.mdx [hunk 2]
```diff
@@ -365,13 +365,14 @@ To achieve consistent usage of translation values and reduce redundancy, you can
 
 ```tsx filename="i18n.ts"
```
Comment (amannn) [inline docs/pages/docs/usage/configuration.mdx:366]:
```suggestion
```tsx filename="i18n.tsx"
```

The comment from @A7med3bdulBaset below is correct, thanks for chiming in!

**Review by amannn — COMMENTED** [review]
Thanks for opening this PR and starting the discussion!

File: docs/pages/docs/usage/configuration.mdx [hunk 3]
```diff
@@ -363,15 +363,16 @@ To achieve consistent usage of translation values and reduce redundancy, you can
 <Tabs items={['i18n.ts', 'Provider']}>
 <Tab>
 
-```tsx filename="i18n.ts"
+```tsx filename="i18n.tsx"
 import {getRequestConfig} from 'next-intl/server';
+import { createComponent, ReactNode } from 'react;
```
Comment (amannn) [inline docs/pages/docs/usage/configuration.mdx:368]:
```suggestion
import {getRequestConfig} from 'next-intl/server';
```

File: docs/pages/docs/usage/configuration.mdx [hunk 4]
```diff
@@ -363,15 +363,16 @@ To achieve consistent usage of translation values and reduce redundancy, you can
...
 
 export default getRequestConfig(async ({locale}) => {
   // ...
 
   return {
     defaultTranslationValues: {
-      important: (chunks) => <b>{chunks}</b>,
+      important: (chunks) => createComponent('b', null, chunks) as ReactNode,
```
Comment (amannn) [inline docs/pages/docs/usage/configuration.mdx:375]:
```suggestion
      important: (chunks) => <b>{chunks}</b>,
```

**Pull request merged**